apps/apollo-stories/src/pages/Coordonnees/Coordonnees.tsx

```diff
@@ -41,7 +41,7 @@ export const Coordonnees = () => {
 
   return (
     <>
-      <DebugGrid />
+      <DebugGrid isCheckedByDefault />
```

Comment:
il y a un props caché pour faire apparaitre le DebugGrid dans un build de production c'est: `forceVisible`

```suggestion
      <DebugGrid isCheckedByDefault forceVisible />
```